ansible/ansible-deployment-example.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: localhost
+
+  collections:
+    - kubernetes.core
+
+  vars:
+    awx_namespace: awx
+    service_type: ClusterIP
+    ingress_type: Ingress
+    admin_user: admin
+    admin_email: "admin@localhost"
+
+  tasks:
+    - name: Deploy AWX Operator
+      include_tasks: deploy-awx-operator-kustomization.yml
```

Comment:
We could add an extra variable called `operator_only=true`, and have a `when:` here.  

Same for the AWX CR below.